### PR TITLE
Create interface in firebase.ts

### PR DESCRIPTION
**Pull request by jeffplays2005**

### CODE REVIEW

**Comment by jeffplays2005** [comment]
Essentially just an initial concept for the event booking slots, has room for improvement and expenditures.

File: server/src/data-layer/models/firebase.ts [hunk 1]
```diff
@@ -129,3 +129,36 @@ export interface BookingChange {
...
+  /**
+   * The location of this event
+   */
+  location: string
+  /**
+   * The start date and end date of this event.
+   * Note that this date is UTC time.
+   * Use the same start and end date to show that its a 1 day event.
+   */
+  start_date: Timestamp
```
Comment (bcho892) [inline server/src/data-layer/models/firebase.ts:157]: each of these need to be individually annotated

File: server/src/data-layer/models/firebase.ts [hunk 2]
```diff
@@ -129,3 +129,36 @@ export interface BookingChange {
...
+  /**
+   * The title of this event
+   * @example Snowboarding race!
+   */
+  title: string
+  /**
+   * An optional description for this event
+   * @example Come join your friends in this exciting UASC event and race!
+   */
+  description?: string
```
Comment (bcho892) [inline server/src/data-layer/models/firebase.ts:143]: maybe mention this should be markdown?

**Review by bcho892 — APPROVED** [review]
happy for merge if wiki is updated

**Comment by jeffplays2005** [comment]
Wiki created 

https://github.com/UoaWDCC/uasc-web/wiki/Firestore-Architecture#event_slot-collection

**Pull request merged**